rom/laghos.cpp

```diff
@@ -1687,20 +1773,29 @@ int main(int argc, char *argv[])
         if(usingWindows && romOptions.parameterID == -1) outfile_twp.close();
     }
 
+    double relative_error = -1;
+
+    if (rom_build_database && rom_calc_rel_error)
+    {
+        relative_error = PrintDiffParGridFunction(normtype, myid, outputPath + "/Sol_Position" + "_" + to_string(romOptions.blast_energyFactor), x_gf);
```

Comment:
What solutions are being compared here (how were they computed and at what parameters)?

Reply:
The greedy algorithm told us to make a new ROM at point B because the residual was high, and it is predicting that the relative error at point B will be greater than 0.1 (or some other threshold). So we create a new ROM at point B. We now want to check whether the greedy algorithm made a correct prediction.

We use the ROM made at point A to run an online_sim at point B. We compare the lifted ROM solution at point B to the FOM solution at point B (which we have already saved from a previous run) to get the relative_error.